cachi2/core/package_managers/rpm/main.py

```diff
@@ -319,6 +319,16 @@ def _download(lockfile: RedhatRpmsLock, output_dir: Path) -> dict[Path, Any]:
             }
             Path.mkdir(dest.parent, parents=True, exist_ok=True)
 
+        for md in arch.metadata:
```

Comment:
This function was refactored, could you please update your change to match the new code?